ListLinked/LinkedList.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+
+namespace LinkedList
+{
+    public  class LinkedList<T> : IEnumerable<T>
+    {
+        private int position = 0;
+        public Node<T> head;
```

Comment:
The general recommendation is that fields shouldn't be public. Use properties instead. But in this case head variable looks like a private member. So please make it private if it really is.